utils/internal/download_gigaspeech_from_tsinghua.sh

```diff
@@ -80,7 +76,7 @@ fi
 download_object_from_release() {
   local obj=$1
   echo "$0: Downloading $obj"
-  local remote_obj=${GIGASPEECH_RELEASE_URL}/$obj
+  local remote_obj=http://www.tsinghua-ieit.com/dataset/GigaSpeech/$obj
```

Comment:
This is the same as my above comment.  Try to abstract our release as a uniform resource idendifier, representing a remote directory, which contains the standard content of GigaSpeech dataset.  Don't assume what form of the URI is(it may be a oss/ftp/http address), and config these concrete addresses in a centralized place/resource.